internal/provisioner/kops_provisioner_cluster_installation.go

```diff
@@ -71,16 +66,22 @@ func (provisioner *KopsProvisioner) CreateClusterInstallation(cluster *model.Clu
 			},
 		},
 		Spec: mmv1alpha1.ClusterInstallationSpec{
-			Size:                   installation.Size,
-			Version:                translateMattermostVersion(installation.Version),
-			IngressName:            installation.DNS,
-			UseServiceLoadBalancer: true,
-			MattermostEnv:          installation.MattermostEnv.ToEnvList(),
-			ServiceAnnotations: map[string]string{
-				"service.beta.kubernetes.io/aws-load-balancer-backend-protocol":        "tcp",
-				"service.beta.kubernetes.io/aws-load-balancer-ssl-cert":                *certificateSummary.CertificateArn,
-				"service.beta.kubernetes.io/aws-load-balancer-ssl-ports":               "https",
-				"service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout": "120",
+			Size:          installation.Size,
+			Version:       translateMattermostVersion(installation.Version),
+			IngressName:   installation.DNS,
+			MattermostEnv: installation.MattermostEnv.ToEnvList(),
+			UseIngressTLS: true,
+			IngressAnnotations: map[string]string{
```

Comment:
Since data is passing through a ALB, can you enable compression here? You could perhaps just use the defaults for gzip-level, gzip-min-length and gzip-types.

https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#gzip-level

Reply:
@gsagula thanks good idea. I enabled gzip which will apply the default values for level, length  and types.